cypress/features/regression/designSystem/simpleSelect.feature

```diff
@@ -108,4 +108,56 @@ Feature: Design System Select component
     Examples:
       | position | size  |
       | bottom   | large |
-      | top      | small |
\ No newline at end of file
+      | top      | small |
+
+  @positive
+  Scenario: Check the onOpen, onClick, onFocus after clicking on the input
+    Given I open "Design System Select" component page "default story" in no iframe
+    When I click on default Select input
+    Then onOpen action was called in Actions Tab
+      And onFocus action was called in Actions Tab
+      And onClick action was called in Actions Tab
+
+  @positive
+  Scenario: Check the onChange event by clicking mouse on the select list option
+    Given I open "Design System Select" component page "default story" in no iframe
+      And I click on default Select input
+    When I click on "first" option on Select list
+    Then onChange action was called in Actions Tab
+
+  @positive
+  Scenario: Check the onKeyDown event after clicking arrow
+    Given I open "Design System Select" component page "default story" in no iframe
+      And I focus default Select input
+    When I click onto default select using "downarrow" key
+      And I wait 500
+    Then onKeyDown action was called in Actions Tab
+      And onFocus action was called in Actions Tab
+
+  @positive
+  Scenario Outline: Check the onKeyDown event after press <key>
+    Given I open "Design System Select" component page "default story" in no iframe
+      And I focus default Select input
+    When I click onto default select using "<key>" key
+      And I wait 500
+    Then onOpen action was called in Actions Tab
+      And onKeyDown action was called in Actions Tab
+      And onFocus action was called in Actions Tab
+    Examples:
+      | key   |
+      | Enter |
+      | Space |
+
+  @positive
+  Scenario: Check the onBlur event
+    Given I open "Design System Select" component page "default story" in no iframe
+      And I focus default Select input
+    When I click on Select label
+    Then onBlur action was called in Actions Tab
+
+  @positive
+  Scenario: Close Select list using Esc keyboard
+    Given I open "Design System Select" component page "default story" in no iframe
+      And I click on default Select input
+    When I hit ESC key in noIframe
```

Comment:
If the page is already opened in no iFrame, does this key depression need to be specified in noIframe here?

Reply:
it will require some more refactor :) but as you wish :)